packages/runtime/src/builder/serialization/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './control-serialization'
-export * from './function-serialization'
+export { serializeControls, deserializeControls } from './control-serialization'
 export type { Serialize, Deserialize } from './controls/types'
+export type { SerializedControl } from './types'
```

Comment:
I went ahead and only exported what was needed, but @migueloller let me know if I am missing an export that is being used in the builder.